.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,29 @@
+// For format details, see https://aka.ms/vscode-remote/devcontainer.json or this file's README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.112.0/containers/cpp
+{
+	"name": "C++",
+	"dockerFile": "Dockerfile",
+	// Due to a limitation in podman, the development container should be executed
+	// in "privileged" mode: https://github.com/containers/libpod/issues/4131
+	"runArgs": [
+		"--privileged"
+	],
+	// "runArgs": [ "--cap-add=SYS_PTRACE", "--security-opt", "seccomp=unconfined" ],
+	// Set *default* container specific settings.json values on container create.
+	"settings": {
+		"terminal.integrated.shell.linux": "/bin/bash"
+	},
+	// Expose container ports
+	"forwardPorts": [
+		2222
+	],
+	// IDs of extensions to install inside the container
+	"extensions": [
+		"ms-vscode.cpptools",
+		"twxs.cmake",
+		"eamodio.gitlens",
+		"github.vscode-pull-request-github",
+		"ms-python.python",
+		"redhat.vscode-yaml"
+	]
+}
```

Comment:
On a lot of files, there isn't a newline at the end of the file.
I remember it was written somewhere that it is required, but I couldn't find it.

Reply:
JSON linter removes empty lines from end-of-files.
We have this in CPP files, but not sure it should apply to JSON settings files as well.